.github/workflows/purge-test-resources.yaml

```diff
@@ -93,14 +96,16 @@ jobs:
               echo " * :white_check_mark: $name - creationTime: $creation_time"  >> $GITHUB_STEP_SUMMARY
             fi
           done <<< "$resource_groups"
+
       - name: Delete Azure resource groups
         run: |
           echo "## Deleting resource group list" >> $GITHUB_STEP_SUMMARY
           cat ${{ env.AZURE_RG_DELETE_LIST_FILE}} | while read line
           do
               echo " * $line" >> $GITHUB_STEP_SUMMARY
-              az group delete --resource-group $line --yes --verbose --no-wait
```

Comment:
Removing the `--no-wait` because we are running into a throttling issue. GitHub blocks our requests because we are sending too many deletion requests.

Reply:
Why does this help? And why is Github involved in a call we make the Azure API?

Reply:
My bad. Ignore this comment please :) I will update this PR.